### DISCUSSION TITLE
Is it possible to add a fade in transition between two scenes ?

### DISCUSSION THREAD

**Discussion by Helyios** [naver/egjs-view360#413] [Q&A]
Hi everyone,

Is it possible to add a fade in transition between two scenes ? I need to have 3 separate scenes and I need to make the new scene appear on a fade-in transition. Is there any way to do so ?

Thank you in advance

**Comment by WoodNeck** [comment]
Hello @Helyios
View360 doesn't support transition yet, but it's planned.
For now, scenes will be changed immediately like in [the hotspot demo](https://naver.github.io/egjs-view360/demo/hotspot).
Contributions are welcomed!
